### ISSUE TITLE
Paye (Manquement)

### ISSUE THREAD

**Issue by Floresmm** [formationinstech/formation-etudiant-2020#10]
Après avoir reçu le premier talon de chèque, j'ai constaté que j'avais été payé seulement 1 journée. (Je ne sais pas si tout le monde à le même problème) Bref, j'aimerais savoir si c'est normal ou si on sera payé la journée qui nous manque lors de la prochaine paye. 

Merci!

**Comment by cedrickcote** [comment]
Je trouve ça étrange aussi, puisque je voulais recourir à l'aide financière du gouvernement. Mais si nous sommes payé dans la prochaine paye, les dates ne seront pas les bonnes, donc je n'aurai pas la possibilité de faire la demande.

**Comment by massimoetpapa** [comment]
Le mot de passe c'est quoi exactement?

**Comment by Floresmm** [comment]
TCI en majuscule


Télécharger Outlook pour Android<https://aka.ms/ghei36>
________________________________

**Comment by massimoetpapa** [comment]
Ouin du 15-28 on a travaillé 2 fois... normalement on serais censé avoir le double

**Comment by formationinstech** [comment]
Oui, l'autre journée va être payé la prochaine paie, parce que vous  avez commencé au milieu d'une période de paie

Le 1 avr. 2020 18 h 07, Massimo Caputo <notifications@github.com> a écrit :

Ouin du 15-28 on a travaillé 2 fois... normalement on serais censé avoir le double

-

**Comment by formationinstech** [comment]
Le deuxième dimanche tu t'es présenté  à Laval ou à Gatineau?

Le 4 avr. 2020 13 h 08, formation1 instech <formationinstechtel@transelec.com> a écrit :
Oui, l'autre journée va être payé la prochaine paie, parce que vous  avez commencé au milieu d'une période de paie

Le 1 avr. 2020 18 h 07, Massimo Caputo <notifications@github.com> a écrit :

Ouin du 15-28 on a travaillé 2 fois... normalement on serais censé avoir le double

-
